nevergrad/benchmark/optgroups.py

```diff
@@ -192,6 +192,10 @@ def all_bo() -> tp.Sequence[Optim]:
     return sorted(x for x in ng.optimizers.registry if "BO" in x)
 
 
+def all_nlopts() -> tp.Sequence[Optim]:
+    return sorted(x for x in ng.optimizers.registry if "NLOPT" in x)
```

Comment:
return [NonObjectOptimizer("NLOPT_" + x).set_name("NLOPT_" +x) for x in ["LN_SBPLX"...]]
this way, no more need to register

Reply:
Done.